frontend/src/layout/navigation/navigationLogic.ts

```diff
@@ -20,62 +22,107 @@ type WarningType =
 
 export const navigationLogic = kea<navigationLogicType<WarningType>>({
     path: ['layout', 'navigation', 'navigationLogic'],
+    connect: {
+        values: [featureFlagLogic, ['featureFlags']],
+    },
     actions: {
-        setMenuCollapsed: (collapsed: boolean) => ({ collapsed }),
-        collapseMenu: () => {},
-        setSystemStatus: (status: SystemStatus) => ({ status }),
-        setToolbarModalOpen: (isOpen: boolean) => ({ isOpen }),
-        setPinnedDashboardsVisible: (visible: boolean) => ({ visible }),
-        setInviteMembersModalOpen: (isOpen: boolean) => ({ isOpen }),
+        toggleSideBar: true,
+        hideSideBar: true,
+        hideAnnouncement: true,
+        openSitePopover: true,
+        closeSitePopover: true,
+        toggleSitePopover: true,
+        showInviteModal: true,
+        hideInviteModal: true,
+        showCreateOrganizationModal: true,
+        hideCreateOrganizationModal: true,
+        showCreateProjectModal: true,
+        hideCreateProjectModal: true,
+        showToolbarModal: true,
+        hideToolbarModal: true,
+        toggleProjectSwitcher: true,
+        hideProjectSwitcher: true,
         setHotkeyNavigationEngaged: (hotkeyNavigationEngaged: boolean) => ({ hotkeyNavigationEngaged }),
-        setProjectModalShown: (isShown: boolean) => ({ isShown }),
-        setOrganizationModalShown: (isShown: boolean) => ({ isShown }),
     },
     reducers: {
-        menuCollapsed: [
-            typeof window !== 'undefined' && window.innerWidth <= 991,
+        isSideBarShownRaw: [
+            window.innerWidth >= 992, // Sync width threshold with Sass variable $lg!
```

Comment:
Any sense to put this in [`windowValues`](https://keajs.org/docs/plugins/window-values/)?

Reply:
Oh, didn't know about `windowValues`! Though it seems this doesn't work for a reducer's default state. I'll definitely use it when resolving #7091